ChefExtensionHandler/bin/chef-install.psm1

```diff
@@ -51,7 +51,7 @@ function Install-ChefClient {
   $completed = $false
 
   while (-not $completed) {
-    echo "Downloading Chef Client ..."
+    echo "Checking Chef Client ..."
```

Comment:
We need add echo statements for each of the scenario here. We should be showing Downloading message when we are doing that process.